js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -188,6 +189,7 @@ export const ASSET_NODE_LIVE_FRAGMENT = gql`
 export const ASSET_NODE_FRAGMENT = gql`
   fragment AssetNodeFragment on AssetNode {
     id
+    ...AssetNodeConfigFragment
```

Comment:
We're planning on moving the data required at execution time out to a separate (deferred) GraphQL request and I think this makes it even more urgent. ATM we fetch ASSET_NODE_FRAGMENT for every asset node on the entire instance as soon as you open the asset catalog.